payment/app.py

```diff
@@ -95,6 +94,17 @@ def watch(service_id):
     return jsonify(), 200
 
 
+@app.route('/whitelist', methods=['POST'])
+def whitelist():
+    print(request.get_json())
+    print(type(request.get_json()))
```

Comment:
use logger

Reply:
Only used it for local debugging, I will remove it